ui/src/app/views/workspace/component.html

```diff
@@ -31,17 +31,12 @@ <h2>{{workspace.name}}</h2>
   {{workspace.description}}
 </div>
 <div class="area-block">
-  <h3>CDR Version</h3>
-  <div class="indented">
-    <p class="cdr-text">
-      {{workspace.cdrVersionId}}
-    </p>
-  </div>
+  <h3 class="cdr-text">CDR Version {{workspace.cdrVersionId}}</h3>
 </div>
 <div class="area-block">
   <div class="table-header">
     <h3 class="table-header-descriptor">Cohorts</h3>
-    <a class="btn btn-link" [routerLink]="['cohorts', 'build']">Add a Cohort</a>
+    <a class="btn btn-link" [ngClass]="{'awaiting-review' : awaitingReview}" (click)="buildCohort()">Add a Cohort</a>
```

Comment:
fyi if its just a single class you can write `[class.awaiting-review]="awaitingReview"` instead (no need to change anything here just spreading the knowledge).